app/views/doorkeeper_applications/access_logs.html.erb

```diff
@@ -36,6 +20,32 @@
                })
 %>
 
+<form>
+<%= render "govuk_publishing_components/components/checkboxes", {
+  name: "include_smokey_users",
+  items: [
+    {
+      label: "Include Smokey Users",
+      value: "true",
+      checked: params["include_smokey_users"] == "true"
+    }
+  ]
+} %>
+<%= render "govuk_publishing_components/components/input", {
+  label: {
+    text: "Month"
```

Comment:
Is "month" clear enough here for users, or do we want to bake "Year" into the text at all if we're requiring that too? This could apply to the params/name etc too but I think that's less important.

Reply:
Yeah - "Year and month" is better I think. The hint text has the format, so hopefully together that will be clear enough.